stock_cycle_count/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,59 @@
+<odoo>
+    <record id="res_config_settings_view_form_cycle_adjustment" model="ir.ui.view">
+        <field
+            name="name"
+        >res.config.settings.view.form.inherit.cycle.adjustment</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="stock.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div name="barcode_setting_container" position="after">
+                <h2>Cycle Count</h2>
+                <div
+                    class="row mt16 o_settings_container"
+                    name="cycle_count_setting_container"
+                >
+                    <div
+                        class="col-12 col-lg-6 o_setting_box"
+                        id="cycle_count_id"
+                        title="Cycle Count"
+                    >
+                        <div class="o_setting_left_pane">
+                            <field
+                                name="auto_start_inventory_from_cycle_count"
+                                widget="upgrade_boolean"
```

Comment:
@DavidJForgeFlow Please remove this widget, because this is what causes it to request to upgrade to Enterprise when activating the setting.

Reply:
Done! Thanks Jordi!